appveyor.yml

```diff
@@ -36,24 +39,35 @@ cache:
 shallow_clone: true
 
 install:
-  # We want  protoc.exe on PATH.
-  - SET PATH=C:/%RIPPLED_DEPS_PATH%;%PATH%
-
   # Download dependencies if appveyor didn't restore them from the cache.
   # Use 7zip to unzip.
   - ps: |
-        if (-not(Test-Path 'C:/$env:RIPPLED_DEPS_PATH')) {
-            echo "Download from $env:RIPPLED_DEPS_URL"
-            Start-FileDownload "$env:RIPPLED_DEPS_URL"
-            7z x "$($env:RIPPLED_DEPS_PATH).zip" -oC:\ -y > $null
-            if ($LastExitCode -ne 0) { throw "7z failed" }
+        if (-not(Test-Path "C:/$env:RIPPLED_DEPS_PATH/openssl")) {
```

Comment:
I don't feel strongly about this, but checking for just one of the expected dirs seems weird. How about using that version file referenced below? Or possibly create an all_done file after the file extractions and look for that? If none of these suggestions make sense, I'm fine with it as is.

Reply:
Yeah, that makes sense. I used `openssl` because it is the last item on the list to be downloaded and extracted, so I was in a sense treating it as an `all_done` file. The version file, specifically, doesn't really work because each archive has one, so I'd have to check for the last item there, too.

Because Appveyor doesn't cache on failures, it might actually be sufficient to only check for `RIPPLED_DEPS_PATH`, and make sure we update that path any time the dependencies change (which is the intention anyway). Thoughts?

Reply:
unless it pre-creates the cache path, checking just for that seems fine. I'm fine with whatever you decide.